apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -287,24 +365,25 @@ type FailedResourcePlacement struct {
 	Condition metav1.Condition `json:"condition"`
 }
 
-// ResourcePlacementConditionType defines a specific condition of a resource placement.
+// ResourcePlacementConditionType defines a specific condition of the cluster resource placement.
 // +enum
 type ResourcePlacementConditionType string
 
 const (
-	// ResourcePlacementConditionTypeScheduled indicates whether we have selected at least one resource to be placed to at least one member cluster and created work CRs under the corresponding per-cluster namespaces (i.e., fleet-member-<member-name>).
+	// ResourceScheduledConditionType indicates whether we have selected at least one resource to be placed to at least one
+	// member cluster and created work CRs under the corresponding per-cluster namespaces (i.e., fleet-member-<member-name>).
```

Comment:
Does this mean for pickN, Scheduled is true after we schedule to one cluster (instead of N)?